libethereum/BlockQueue.cpp

```diff
@@ -131,6 +127,7 @@ void BlockQueue::verifierBody()
 				}
 			cwarn << "BlockQueue missing our job: was there a GM?";
 			OK1:;
+			drainVerified();
```

Comment:
That would be misleading. The main purpose of this function is to clear the front of m_verifying of the blocks that have been verified, no matter good or bad. Note that is is also used on line 150 for the case of good blocks.